### DISCUSSION TITLE
How to style a <Box> </Box> JSX Element

### DISCUSSION THREAD

**Discussion by falcoda** [MetaMask/snaps#2681] [Q&A] [answered]
Hello, I am trying to add a border to a jsx box, But the props does not allow any styles or classname.
Is there another way?

Thanks you

**Comment by Montoya** [comment]
To ensure that Snaps interfaces are consistent with the rest of MetaMask, we do not offer very much customization. You cannot apply arbitrary CSS on components. We will offer some customization in the future, but for now a Box cannot have any props. You can review all of the syntax and props for all components here: https://docs.metamask.io/snaps/features/custom-ui/with-jsx/

**Reply by falcoda** [reply]
Thanks for you quick response.
In fact the only style that I want is already present in metamask. 

It's this little border present on the transaction confirmation : 
![image](https://github.com/user-attachments/assets/f92cf005-5aba-444b-a7d2-88a1273e866c)

There are no way to access to it? 

Thanks

**Reply by Montoya** [reply, marked as answer]
No, I apologize, that border is part of an older design that we will eventually change to match the new design system, which looks like this: 

<img width="346" alt="sign" src="https://github.com/user-attachments/assets/5945fdbf-d33e-4a1c-b54a-37b458b39df7">

Soon we will introduce a component that gives this rounded border appearance, but for now there is no way to do it.

**Reply by falcoda** [reply]
Ok, thanks you for the information !

**Reply by 0xAskar** [reply]
Is there an update on customization?
